src/stylesheets/core/_utilities.scss

```diff
@@ -26,22 +26,12 @@
 // Unstyled list helper
 
 @mixin unstyled-list() {
-  display: block;
+  @include margin(0 null);
```

Comment:
We had `margin: 0` before, when actually the only margins added in the list was top and bottom. We do not need to declare margin=0 on the left and right.